usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+import static org.mockito.BDDMockito.given;
+
+/**
+ * @author Marc Simó
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(fullyQualifiedNames = {"android.util.Base64"})
+public class PetCollectionsManagerClientTest {
+    private static final StringBuilder STATUS_NO_CONTENT_RETURNED = new StringBuilder("204");
+    private static final Integer STATUS_NO_CONTENT_EXPECTED = 204;
+    private static final String ACCESS_TOKEN = "my-token";
+    private static final String USERNAME = "user";
+    private static final String PET_NAME = "Gustavo";
+    private static final String DATE_1 = "1990-01-08T15:20:30";
+    private static final String DATE_2 = "1995-01-08T15:20:30";
+    private static final String DATE_3 = "1998-01-08T15:20:30";
+    private StringBuilder exerciseCollectionJson;
+    private StringBuilder exerciseDataJson;
+    private List<Exercise> exerciseCollectionList;
+    private ExerciseData exerciseData;
+    private StringBuilder illnessCollectionJson;
+    private StringBuilder illnessDataJson;
+    private List<Illness> illnessCollectionList;
+    private IllnessData illnessData;
+    private StringBuilder mealCollectionJson;
+    private StringBuilder mealDataJson;
+    private List<Meal> mealCollectionList;
+    private MealData mealData;
+    private StringBuilder medicationCollectionJson;
+    private StringBuilder medicationDataJson;
+    private List<Medication> medicationCollectionList;
+    private MedicationData medicationData;
+    private StringBuilder vaccinationCollectionJson;
+    private StringBuilder vaccinationDataJson;
+    private List<Vaccination> vaccinationCollectionList;
+    private VaccinationData vaccinationData;
+    private StringBuilder vetVisitCollectionJson;
+    private StringBuilder vetVisitDataJson;
+    private List<VetVisit> vetVisitCollectionList;
+    private VetVisitData vetVisitData;
+    private StringBuilder washCollectionJson;
+    private StringBuilder washDataJson;
+    private List<Wash> washCollectionList;
+    private WashData washData;
+    private StringBuilder weightCollectionJson;
+    private StringBuilder weightDataJson;
+    private List<Weight> weightCollectionList;
+    private WeightData weightData;
+
+    @Mock
+    private TaskManager taskManager;
+
+    @InjectMocks
+    private PetCollectionsManagerClient client = new PetCollectionsManagerClient();
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Before
+    public void setUp() {
+        // Corregir exercise per ficar coordinates
+        exerciseCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "          \"latitude\":34,\n" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)